### ISSUE TITLE
Mov_text not being removed by plugin

### ISSUE THREAD

**Issue by Gumbyohson** [HaveAGitGat/Tdarr#639]
**Describe the bug**
Mov_text not being removed by "Tdarr_Plugin_MP01_MichPasCleanSubsAndAudioCodecs"

**To Reproduce**
Have an MKV with mov_text subtitles. 
Have "Tdarr_Plugin_MP01_MichPasCleanSubsAndAudioCodecs" set to remove mov_text subtitle codecs.

**Expected behavior**
Subtitle codec should be removed.

**Screenshots**
![image](https://user-images.githubusercontent.com/1866263/170859901-c9044433-2ff3-4fa6-b541-548ad4f9917a.png)
![image](https://user-images.githubusercontent.com/1866263/170859958-1d2c4929-10f5-41e0-ae3c-484e5060dff4.png)

**Please provide the following information:**
- Config files  [can be found in /app/configs/ when using Docker or in the /configs folder next to Tdarr_Updater if not using Docker]
[Tdarr_Node_Config.zip](https://github.com/HaveAGitGat/Tdarr/files/8792942/Tdarr_Node_Config.zip)

- Log files  [can be found in /app/logs/ when using Docker or in the /logs folder next to Tdarr_Updater if not using Docker]
[Tdarr_Node_Log.txt.1.zip](https://github.com/HaveAGitGat/Tdarr/files/8792944/Tdarr_Node_Log.txt.1.zip)

-Worker error [can be found on the 'Tdarr' tab by pressing the 'i' button on a failed item in the staged file section or in the transcode error section at the bottom]
[K_G2bV1myU-log.txt](https://github.com/HaveAGitGat/Tdarr/files/8792946/K_G2bV1myU-log.txt)

**Comment by supersnellehenk** [comment]
It fails on `Tdarr_Plugin_MC93_Migz6OrderStreams`, since the file is wrong to begin with. So it doesn't even get to the removal plugin.

**Comment by Gumbyohson** [comment]
So if I move Tdarr_Plugin_MP01_MichPasCleanSubsAndAudioCodecs to the top it might work? I'll give it a go.

**Comment by supersnellehenk** [comment]
> That's fixed it. Is there anything that can be added to "Tdarr_Plugin_MC93_Migz6OrderStreams" such as a force conform option or something?

The Remux plugin from the same creator has one. The order streams one fails because your file is invalid to begin with, since it's an mkv with `mov_text` streams. `mov_text` are only valid in mp4s. I don't think that a force conform option is in the scope of that order streams plugin.

**Comment by Gumbyohson** [comment]
> > That's fixed it. Is there anything that can be added to "Tdarr_Plugin_MC93_Migz6OrderStreams" such as a force conform option or something?
> 
> The Remux plugin from the same creator has one. The order streams one fails because your file is invalid to begin with, since it's an mkv with `mov_text` streams. `mov_text` are only valid in mp4s. I don't think that a force conform option is in the scope of that order streams plugin.

yeah fair... I think just removing it at the top is going to be the play at this point. I have this library plugin stack filtered for only MKV so it should be fine.
